packages/ui/src/composables/theme.ts

```diff
@@ -0,0 +1,22 @@
+import { BasicColorMode, UseColorModeOptions, useColorMode, useLocalStorage } from '@vueuse/core'
+
+/**
+ * priorites following the desc order
+ */
+export const ThemeStroageKeys = {
+  vitepress: 'vitepress-theme-appearance',
+  default: 'vueuse-color-scheme',
+}
+
+export function useTheme<T extends string = BasicColorMode>(options?: UseColorModeOptions<T>) {
```

Comment:
I guess you should find a way to detect the environment. These two keys might both exist.

Reply:
`Array.prototype.find()` follows the asc order to find the target item. So with the `Object.values(ThemeStorageKeys)`, the priority of `vitepress-theme-appearance` is higher than `vueuse-color-scheme`.
However, finding a better way to detect the environment is hard. I will try.